src/Symfony/Component/Translation/Loader/XliffFileLoader.php

```diff
@@ -185,4 +176,35 @@ private function getXmlErrors($internalErrors)
 
         return $errors;
     }
+
+    /**
+     * @param \SimpleXMLElement $noteElement
+     * @param string            $encoding
+     *
+     * @return array
+     */
+    private function setNotesMetadata($noteElement = null, $encoding = null)
```

Comment:
What name do you suggest? What about  `parseNotesMetadata` or `getNotesMetadata` ?